src/GlobalStyle.js

```diff
@@ -11,12 +11,19 @@ body {
   font-size: 16px;
   margin: 0;
   padding: 0;
+  display:flex;
+  align-items: center;
+  justify-content: center;
   font-family: 'Roboto', sans-serif;
   font-weight: 400;
 }
 
 main{
  background: ${props => props.theme.main};
+display: flex;
```

Comment:
The main is also in the body, so wouldn't be defining the style for the body itself be enough ?

Reply:
No, flexbox only affects the child-element.